### PR TITLE
LESS is more

### PR DESCRIPTION
**Pull request by rusackas**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [x] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
This shouldn't change anything in the processed CSS. 

Now that we have LESS everywhere, this PR just goes through and takes advantage of it, tidying up various things readability/grouping

I went through and did an `npm run build` on the `master` branch and this feature branch. I compared all resulting CSS files, and screenshots of the (innocuous as far as I can see) diffs are below - it looks like things were just re-ordered a bit:

image

image
(this yields the same CSS attributes without repeating the properties)

image

image

image

image

image

### CODE REVIEW

**Comment by craig-rueda** [comment]
LGTM, but prob needs to be tested. This definitely makes things easier to read...

File: superset/assets/src/SqlLab/main.less [hunk 1]
```diff
@@ -321,10 +321,9 @@ div.Workspace {
 
 div.tablePopover {
   opacity: 0.7 !important;
-}
-
-div.tablePopover:hover {
-  opacity: 1 !important;
+  &:hover {
```
Comment (etr2460) [inline superset/assets/src/SqlLab/main.less]: this is a styling thing, but personally i prefer a new line between normal styles and the `&:foo {` line. I think there's a way to lint for that, if so maybe you could add that rule in another PR. if not, maybe we could start standardizing that here if you agree?

File: superset/assets/src/dashboard/stylesheets/components/column.less [hunk 2]
```diff
@@ -28,7 +28,6 @@
 
 .dashboard--editing {
   .grid-column:after {
-    border: 1px solid transparent;
```
Comment (etr2460) [inline superset/assets/src/dashboard/stylesheets/components/column.less]: lol, what was this?

**Review by etr2460 — COMMENTED** [review]
A few comments, but otherwise seems legit. could you add a bunch of before and after screenshots to your test plan to confirm that this works. alternatively, you could attach a diff on the processed CSS to confirm it's all correct

File: superset/assets/src/SqlLab/main.less (hunk 1 above)
Comment (rusackas) [reply superset/assets/src/SqlLab/main.less]: I prefer that too, but glad you caught ones I missed. I'll look for a linting rule.

File: superset/assets/src/dashboard/stylesheets/components/column.less (hunk 2 above)
Comment (rusackas) [reply superset/assets/src/dashboard/stylesheets/components/column.less]: There are a bunch of fairly convoluted styles around the dashed borders in the dashboard editor, which seem like they could use a simplification effort (a future PR, probably). I'll double check that this one wasn't providing some useful 1px offset.

**Comment by rusackas** [comment]
> You could attach a diff on the processed CSS to confirm it's all correct.

I like this idea... will do later today.

**Comment by rusackas** [comment]
@etr2460 I pasted a bunch of diff screenshots above. To get these, I ran the build on master, and on this branch. I then took all the resulting CSS files from each build, removed the annoying hashes from the filenames, ran prettier on them so I could get a reasonable DIFF, and then got the DIFF via github. Hopefully I didn't screw anything up in that process, but I'm feeling pretty good about it overall :D

**Review by etr2460 — APPROVED** [review]
Thanks for all the diligence in testing and ensuring the outputted CSS is correct. And double thanks for adding the lint rule!

**Pull request merged**